release/ray_release/buildkite/step.py

```diff
@@ -121,13 +121,16 @@ def get_step(
     if test.get("run", {}).get("type") == "client":
         step["agents"]["queue"] = str(RELEASE_QUEUE_CLIENT)
 
-    # If a test is not stable, allow to soft fail
+    # If a test is not stable or jailed, allow to soft fail
     stable = test.get("stable", True)
-    if not stable:
+    jailed = test.get("jailed", True)
```

Comment:
Should this be the inverse?
```suggestion
    jailed = test.get("jailed", False)
```

Reply:
Nice catch!